### PR TITLE
WIP - CNV preview build for final ack

### PR DESCRIPTION
**Pull request by aburdenthehand**
Adding 2.4 release notes to master branch strictly for a reasonable preview build. This is not to be merged.

### CODE REVIEW

File: virt/virt-2-4-release-notes.adoc [hunk 1]
```diff
@@ -1,11 +1,206 @@
...
+value by running `oc describe node <node>` for all nodes and looking at the
+`cpu-model-<name>` labels. Select the CPU model that is present on all of your
+nodes.
+
+//Jira story for the following content https://issues.redhat.com/browse/CNV-4757. Nelli noted that for CNV 2.5, this can be removed from Known Issues. The defect status, however, is CLOSED.
+* {VirtProductName} cannot reliably identify node drains that are triggered by
+running either `oc adm drain` or `kubectl drain`. Do not run these commands on
+the nodes of any clusters where {VirtProductName} is deployed. The nodes might not
+drain if there are virtual machines running on top of them.
++
```
Comment (nellyc) [inline virt/virt-2-4-release-notes.adoc:205]: I would avoid this additional empty line

File: virt/virt-2-4-release-notes.adoc [hunk 2]
```diff
@@ -1,11 +1,206 @@
...
+
+//CNV-5371 - Import existing virtual machines from Red Hat Virtualization to OpenShift Virtualization.
+* You can import a single Red Hat Virtualization virtual machine by using the virtual machine wizard or the CLI.
+
+//CNV-5463 - New API Subgroup for Virt
+* Every component in {VirtProductName} now uses its own API subgroup, `<component_name>.kubevirt.io`.
+
+[id="virt-2-4-known-issues"]
+== Known issues
+//https://bugzilla.redhat.com/show_bug.cgi?id=1855067 - Requested doc text for this defect.
```
Comment (nellyc) [inline virt/virt-2-4-release-notes.adoc:120]: seems to be covered in line 136

File: virt/virt-2-4-release-notes.adoc [hunk 3]
```diff
@@ -1,11 +1,206 @@
...
+//https://bugzilla.redhat.com/show_bug.cgi?id=1835426 - Requested doc text for this defect.
```
Comment (nellyc) [inline virt/virt-2-4-release-notes.adoc:121]: I believe this one can be removed - the bug was fixed in 2.4.0

File: virt/virt-2-4-release-notes.adoc (hunk 1 above)
Comment (aburdenthehand) [reply virt/virt-2-4-release-notes.adoc:205]: It's keeping the next para indented properly.
Perhaps we should use a sub-bullet like with the other workaround text?

**Comment by aburdenthehand** [comment]
@nellyc Thanks for the reviews!
The feedback was added to #23888 as this one will be closed (only opened to have build for release notes and doc)

**Pull request closed without merging**